--- rspec-core-2.12.2/spec/spec_helper.rb.fedora	2013-01-02 14:26:16.000000000 +0900
+++ rspec-core-2.12.2/spec/spec_helper.rb	2013-01-02 14:31:47.000000000 +0900
@@ -17,7 +17,7 @@
 Spork.prefork do
   require 'rspec/autorun'
   require 'autotest/rspec2'
-  require 'aruba/api'
+  #require 'aruba/api'
 
   if RUBY_PLATFORM == 'java'
     # Works around https://jira.codehaus.org/browse/JRUBY-5678
@@ -101,9 +101,11 @@
     c.alias_it_behaves_like_to 'it_has_behavior'
     c.around {|example| sandboxed { example.run }}
     c.include(RSpecHelpers)
+'''
     c.include Aruba::Api, :example_group => {
       :file_path => /spec\/command_line/
     }
+'''
 
     # runtime options
     c.treat_symbols_as_metadata_keys_with_true_values = true
